src/Modules/Backend/templates/base/head.html.twig

```diff
@@ -13,11 +13,11 @@
 
   {{ encore_entry_script_tags('Backend/js/Backend', null, 'BackendConfig') }}
   {{ encore_entry_link_tags('Backend/css/screen', null, 'BackendConfig') }}
-  {% for jsFile in jsFiles %}
-    <script src="/{{ jsFile }}"></script>
+  {% for js_file in jsFiles %}
```

Comment:
Shouldn't `jsFiles` be changed to `js_files`?

Reply:
fixed a6686b780bd2d77164dc63dc5e37f35340f63da4